src/test/java/seedu/address/logic/commands/ServeCommandTest.java

```diff
@@ -43,4 +45,18 @@ public void execute_borrowerIdNotFound_throwsCommandException() {
         Assert.assertThrows(CommandException.class, MESSAGE_NO_SUCH_BORROWER_ID, () ->
                 serveCommand.execute(modelManager));
     }
+
+    @Test
+    public void execute_alreadyInServeMode_throwsCommandException() throws CommandException {
+        Model modelManager = new ModelManager();
+        new RegisterCommand(BOB).execute(modelManager);
+        new RegisterCommand(ALICE).execute(modelManager);
+
+        BorrowerId validIdBob = new BorrowerId(VALID_ID_BOB);
```

Comment:
I think we should just use `BOB.getBorrowerId()` here instead of using another constant and assuming it will be the same Id as bob

Reply:
Good idea!